### DISCUSSION TITLE
Option: Send update immediately when light is turned on.

### DISCUSSION THREAD

**Discussion by IIV-NATHAN-VII** [basnijholt/adaptive-lighting#743] [Ideas]
So I have smart bulbs that automatically turn on a warm colour to avoid a dazzling transition from white of a night time if set up the other way around.

Could you add an option so when Adaptive Lighting see's a light come online it sends an update to it immediately? instead of having to wait for the update interval?

**Comment by basnijholt** [comment]
That’s what it’s supposed to be doing since 1.19. 

However if you have separate_turn_on_commands turned on it will first immediately do brightness and then later color.

**Reply by IIV-NATHAN-VII** [reply]
@basnijholt  I'm finding some of my bulbs change within like 2 seconds of being powered on while others take around 10, all the same brand. Thought that'd be the reason but I did just update so I'll see how it goes.

**Reply by basnijholt** [reply]
If it doesn’t work in the new version please let me know and share debugging logs.
